support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -53,14 +51,18 @@ class CustomActionBuilders(
   }
 
   private def maybeAuthenticated(onUnauthenticated: RequestHeader => Result): ActionBuilder[OptionalAuthRequest, AnyContent] =
-    new AuthenticatedBuilder(authenticatedIdUserProvider.andThen(Some.apply), cc.parsers.defaultBodyParser, onUnauthenticated)
+    new AsyncAuthenticatedBuilder(
+      userinfo = requestHeader => asyncAuthenticationService.authenticateUser(requestHeader).map(Some.apply),
+      cc.parsers.defaultBodyParser,
+      onUnauthenticated
```

Comment:
funny to name the first parameter but not the rest, either all or none would be worth considering